tests/integration/test_securitydata.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from datetime import timedelta
+from shlex import split as split_command
+
+import pytest
+from tests.integration.util import DataServer
+
+from code42cli.main import cli
+
+
+begin_date = datetime.utcnow() - timedelta(days=20)
+begin_date_str = begin_date.strftime("%Y-%m-%d")
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
```

Comment:
We should split this up into individual tests

Reply:
resolving this conversation as it is being discussed [here](https://github.com/code42/code42cli/pull/212#discussion_r567613905)